client/platform/web-girder/views/Upload.vue

```diff
@@ -83,11 +100,15 @@ export default defineComponent({
           const name = processed.fullList.length === 1 ? processed.fullList[0].name : '';
           preUploadErrorMessage.value = null;
           try {
+            if (dstype !== 'zip') {
             // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
Sorry can we do this one too?  I don't know why these got in here, I'm sure there was a reason.